components/profile/ProfileDetail.vue

```diff
@@ -199,6 +211,9 @@ import { chainsWithMintInteraction } from '@/composables/collectionActivity/help
 import { Interaction } from '@kodadot1/minimark/v1'
 import CollectionFilter from './CollectionFilter.vue'
 import GridLayoutControls from '@/components/shared/GridLayoutControls.vue'
+import ProfileEmptyResult from '@/components/profile/ProfileEmptyResult.vue'
+import { CHAINS, type Prefix } from '@kodadot1/static'
```

Comment:
should we use `CHAINS` or `chainNames`?

Reply:
It's not related to the chain name here.